.github/actions/cleanup-runner/action.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Cleanup Runner'
```

Comment:
```
Cleanup large tools for build space 49s

Prepare all required actions
Run ./.github/actions/cleanup-runner
Run sudo rm -rf /usr/share/dotnet /usr/local/lib/android /opt/ghc /opt/hostedtoolcache/CodeQL
Total reclaimed space: 0B
Total:	0B
Prepare all required actions
Run ./.github/actions/cleanup-runner
Run sudo rm -rf /usr/share/dotnet /usr/local/lib/android /opt/ghc /opt/hostedtoolcache/CodeQL
Total reclaimed space: 0B
Total:	0B
```

Looking at a few jobs, this adds 50 - 60 seconds of runtime to each job even if it does not reclaim anything. I'm wondering if that's worth it?